shared/src/lib/dialects/firebird.ts

```diff
@@ -21,16 +20,33 @@ function wrapIdentifier(id: string) {
   if (id.includes(" ")) return `"${id}"`;
   return id;
 }
+function escape(value) {
+  if (value === null || value === undefined)
+      return 'NULL';
 
+  switch (typeof(value)) {
+      case 'boolean':
+            return value ? 'true' : 'false';
+      case 'number':
+          return value.toString();
+      case 'string':
+          return "'" + value.replace(/'/g, "''").replace(/\\/g, '\\\\') + "'";
+  }
+
+  if (value instanceof Date)
+      return "'" + value.getFullYear() + '-' + (value.getMonth()+1).toString().padStart(2, '0') + '-' + value.getDate().toString().padStart(2, '0') + ' ' + value.getHours().toString().padStart(2, '0') + ':' + value.getMinutes().toString().padStart(2, '0') + ':' + value.getSeconds().toString().padStart(2, '0') + '.' + value.getMilliseconds().toString().padStart(3, '0') + "'";
```

Comment:
Minor thing, a template string might be easier to read and if nothing else is more the pattern for putting a string together like this

Reply:
Lol so uh. Azmy and I kinda stole this directly from firebord's source code 😂 I'm down to change it, just wanna be careful I don't muck it up